### PR TITLE
updating README for directory structure

### PR DESCRIPTION
**Pull request by oraNod**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -36,33 +37,35 @@ Documentation follows a link:https://redhat-documentation.github.io/modular-docs
...
-* attributes: A common directory for attribute files.
-* assemblies: A directory for all assemblies, each on in its own AsciiDoc file. Assemblies are collections of modules linked together.
-** Assemblies are grouped by platform component. Create a new subdirectory to organize assemblies associated with that component.
-* images: A directory for all images and other digital content.
-* modules: A directory for all modules, each one in its own AsciiDoc file.
-** Modules are grouped by platform component. Create a new subdirectory to organize modules associated with that component.
-* titles: A directory for titles (information units, e.g, install guide, user guide). Contains master.adoc files to be called during builds and symbolic links to directories (modules, etc.) that contain .adoc content.
-** Titles are grouped by platform component or platform-level feature. Create a new subdirectory to organize titles associated with a component.
+** `attributes`: A common directory for attribute files.
+** `assemblies`: A directory for all assemblies, each on in its own AsciiDoc file. Assemblies are collections of modules linked together.
```
Comment (rogrange) [inline README.adoc:52]: "each on in its" should read "each in its own"

File: README.adoc [hunk 2]
```diff
@@ -36,33 +37,35 @@ Documentation follows a link:https://redhat-documentation.github.io/modular-docs
...
+*** Assemblies are grouped by platform component. Create a new subdirectory to organize assemblies associated with that component.
+** `images`: A directory for all images and other digital content.
+** `modules`: A directory for all modules, each one in its own AsciiDoc file.
+*** Modules are grouped by platform component. Create a new subdirectory to organize modules associated with that component.
+** `titles`: A directory for titles (information units, e.g, install guide, user guide). Contains master.adoc files to be called during builds and symbolic links to directories (modules, etc.) that contain .adoc content.
+*** Titles are grouped by platform component or platform-level feature. Create a new subdirectory to organize titles associated with a component.
```
Comment (rogrange) [inline README.adoc:58]: Can we rename this TItles_Platform" or something that distinguishes this from the lower level title manual titles?

Comment (oraNod) [reply README.adoc:58]: Sure, do you want to push changes to my branch?

File: README.adoc [hunk 3]
```diff
@@ -36,33 +37,35 @@ Documentation follows a link:https://redhat-documentation.github.io/modular-docs
...
 ....
 
 Top-level directories are:
 
-* downstream: A directory that stores the Red Hat Ansible Automation platform modules, assemblies, and titles.
+* `common`: Contains modules and attributes that are shared across Red Hat Ansible Automation branches.
+You should modify content in the `common` folder on the `main` branch only.
 
-Lower-level directories are:
+* `downstream`: Contains Red Hat Ansible Automation platform modules, assemblies, and titles in these sub-folders:
```
Comment (rogrange) [inline README.adoc:49]: Platform should be capitalized as it is part of the application name.

**Review by rogrange — COMMENTED** [review]
I found a few errors, nothing major. It looks good.

File: README.adoc (hunk 2 above)
Comment (rogrange) [reply README.adoc:58]: That, and I forgot how to do it.

**Review by rogrange — COMMENTED** [review]
Everything looks good Don. thank you for the name change. I can see myself working quickly and putting the wrong file in the wrong directory.

**Comment by oraNod** [comment]
Thanks @rogrange and @lmalivert Just curious but are you able to approve? I don't mean to come across weird I was just expecting to get approvals rather than comments in the PR. Maybe the team has a different way of working and I need to adjust my expectations though. Cheers.

**Pull request merged**